src/validation.cpp

```diff
@@ -3137,6 +3171,10 @@ bool CheckBlock(const CBlock& block, CValidationState& state, const Consensus::P
     if (nSigOps * WITNESS_SCALE_FACTOR > MAX_BLOCK_SIGOPS_COST)
         return state.DoS(100, false, REJECT_INVALID, "bad-blk-sigops", false, "out-of-bounds SigOpCount");
 
+    if (!MWEB::Node::CheckBlock(block, state)) {
+        return false;
+    }
```

Comment:
No peer banning?

Reply:
That occurs inside of MWEB::Node::CheckBlock

Reply:
Sorry missed that. Might be worth a comment pointing to that in validation if I'm being a dick about it ;)